### ISSUE TITLE
💡 [REQUEST] - Allow the possibilty to link dropsets

### ISSUE THREAD

**Issue by TheSecondWing** [madmustachecompany/Blast-Workout-App#70]
### Summary

Hello,

Would it be possible to allow the user to link dropsets, allowing for a better comprehension when looking at an exercise log?

Thanks.

### Basic Example

I have an exercise log that looks like that:

1. Warm-up set
2. Normal set
3. Dropset 1 part 1
4. Dropset 1 part 2
5. Dropset 2 part 1
6. Dropset 2 part 2

Being able to link in some sort of way sets 3 and 4 together, and sets 5 and 6 together would allow for a better comprehension of how the dropsets were actually decomposed in the session.
Otherwise it just looks like four sets tagged as "dropset" one after another.

**Comment by madmustachecompany** [comment]
I agree that the design of the drop-sets could be improved for better visibility (and ideally we'd like to develop an "auto-calculation" feature for drop and warm-up series).

Meanwhile, here's how I manage it personally:
I always mark the first series of a drop-set as a normal series and the following ones as drop-sets. To take your example again, I'll mark it as follows:
1. Warm-up
2. Normal
3. Normal => set 3 part 1
4. Drop => set 3 part 2
5. Normal => set 4 part 1
6. Drop => set 4 part 2
<img width="153" alt="image" src="https://github.com/madmustachecompany/Blast-Workout-App/assets/120519094/794476e1-d154-458f-8f5d-8c6e0e1386ed">

and with group of 3 drop-sets it looks like this:
<img width="165" alt="image" src="https://github.com/madmustachecompany/Blast-Workout-App/assets/120519094/1e9a12f4-5ab3-4264-aab2-d7d1babdd045">  => 2 series of 3 drop-sets

**Comment by TheSecondWing** [comment]
Thanks for the input. That makes sense, I'll register my dropsets like shown in your comment.
